### PR TITLE
Security Considerations for Unsigned Credential ID

### LINKED ISSUE #394: "credential ID" not signed over by authenticatorGetAssertion operation
I note that in [authenticatorGetAssertion operation](https://w3c.github.io/webauthn/#op-get-assertion), the "credential ID" not signed over -- i.e., it is not included in `authenticator data` because no `attObj` is contained in the `authenticator data` returned by this operation.  As I (vaguely) recall, we discussed this long ago and determined that the worst downside of a buggy authenticator returning an incorrect credential ID is that the RP will not look up the correct cred public key with which to verify the returned signed `authenticator data` (aka assertion) and the overall ceremony would thus end in error. 

we should probably document this rationale and consequence somewhere in the spec.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #394

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/webauthn/pull/766.html" title="Last updated on Jan 26, 2018, 4:21 PM GMT (86f8f7c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/766/1e77b42...selfissued:86f8f7c.html" title="Last updated on Jan 26, 2018, 4:21 PM GMT (86f8f7c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -4427,7 +4427,13 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
 [[FIDOMetadataService]] provides one way to access such information.
 
 
+## credentialId Unsigned ## {#credentialIdSecurity}
 
+The credential ID is not signed.
+This is not a problem because all that would happen if an authticator returns
```
Comment (emlun) [inline index.bs]: Typo: authticator

File: index.bs [hunk 2]
```diff
@@ -4427,7 +4427,13 @@ RogueList when performing ECDAA-Verify (see section 3.6 in [[!FIDOEcdaaAlgorithm
...
+the wrong crednetial ID is that the RP will not look up the correct
```
Comment (emlun) [inline index.bs]: Typo: crednetial
Suggest: RP -> [=[RP] =]

Comment (emlun) [reply index.bs]: I think a more interesting case to point out is if a MitM were to manipulate the credential ID (the conclusion is the same), so I would probably write this as something like "[...] if an authenticator returns the wrong credential ID, or if an attacker intercepts and manipulates the credential ID, is that the RP [...]"

**Comment by emlun** [comment]
Also, perhaps link some of the terms throughout? I can make a PR of that into this PR if you can't run bikeshed.

**Comment by selfissued** [comment]
@emlun if you can create a PR into my PR addressing the issues you raise that would great.  Then I could apply it in my morning and merge the combination into master.

**Pull request merged**